schemas/v1/worker-configuration-result.yml

```diff
@@ -0,0 +1,21 @@
+$schema: http://json-schema.org/draft-06/schema#
+title: "Outcome of a worker configuration evaluation"
+type: object
+properties:
+  workerType:
+    type: string
+  providerData:
+    type: object
+  biddingStrategyData:
+    type: object
+  documentationData:
+    type: object
+  schemaData:
+    type: object
+requireProperties:
```

Comment:
`required`

Also, I think you want `additionalProperties: false` for most of these.